pycloudlib/util.py

```diff
@@ -280,3 +280,13 @@ def _safe_int(possible_int):
         return int(possible_int)
     except (ValueError, TypeError):
         return None
+
+
+def ubuntu_name_to_number(name):
+    """Convert to version number based on the version name."""
```

Comment:
Any objection to a global dict? The function seems a bit over the top of a dict lookup. /me was tempted to suggest distro-info, but that's an external dep dependency. 

UBUNTU_RELEASE_VERSION_MAP = {
    'focal': '20.04',
    'bionic': '18.04',
    'xenial': '16.04',
}

Reply:
Not familiar with distro-info. I'm willing to entertain an external dependency, as that can be automatically updated. I'm not really a big fan of what I did here because it means it will need to be updated after every new release. Global dict makes sense though.

Reply:
No need for this. If it becomes a maintenance burden we can look at using an external dynamic utility for this, but python-proper libraries or modules are probably limited here I think as most modules would want to know "what environment I'm running in" instead of "what is the known set/map of all releases available".